sequins.go

```diff
@@ -56,7 +55,16 @@ func (s *sequins) start(address string) error {
 	s.started = now
 	s.updated = now
 
-	defer s.index.Close()
+	return nil
+}
+
+func (s *sequins) start(address string) error {
+	// TODO: we may need a more graceful way of shutting down, since this will
```

Comment:
is the issue that we should wait for the http server to shut down? in that case we could just switch in https://github.com/icub3d/graceful/. That can be a separate thing, though.